fe/fe-core/src/main/java/org/apache/doris/cloud/qe/CloudCoordinator.java

```diff
@@ -81,13 +92,73 @@ protected void prepare() {
     }
 
     protected void processFragmentAssignmentAndParams() throws Exception {
-        super.processFragmentAssignmentAndParams();
+        prepare();
```

Comment:
override prepare  and move line 96 ~ 100 at the end of CloudCoordinator.end

Reply:
delelte override processFragmentAssignmentAndParams

Reply:
fixed